pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/Bytes.java

```diff
@@ -154,6 +157,34 @@ void writeTo(@NonNull final ByteBuffer dstBuffer) {
         dstBuffer.position(dstBuffer.position() + length);
     }
 
+    /**
+     * Package private helper method for efficient copy of
+     * our data into an OutputStream without creating a defensive copy
+     * of the data. The implementation relies on a well behaved
+     * OutputStream that doesn't modify the buffer data.
+     *
+     * @param outStream the OutputStream to copy into
+     */
+    public void writeTo(@NonNull final OutputStream outStream) {
+        try {
+            outStream.write(buffer);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Can be a UncheckedIOException

Reply:
Fixed.

Reply:
Actually changing to UncheckedIOException gives the following error. Keeping the IOException...
/Users/user/local/pbj/pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/Bytes.java:171: error: unreported exception IOException; must be caught or declared to be thrown
            outStream.write(buffer);

Reply:
I mean UncheckedIOException from java.io rather than RuntimeException
```
 } catch (IOException e) {
             throw new UncheckedIOException(e);
}
```

Reply:
Ah... Got it!